### PR TITLE
Add memo to file entity type

### LINKED ISSUE #962: Add memo field for file entity
Summary
Each file entity should include an optional memo field of up to 100 bytes. Uniqueness is not guaranteed. The memo can only be changed by the admin keys on the file entity.

Memo field would be returned as part of file info.

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Related issue(s)**:
- Closes #962

**Summary of the change**:
- Add `memo` field to `JFileInfo -> HFileMeta` type, update serdes version.
- Add memo syntax checks in file create/update transition logics.
- Respect memo in create/update transition logic.
- Incorporate memo size in fee calculations, refactoring to follow the new estimate pattern. 
- Migrate legacy property `binary.object.query.retry.times` to node-local `queries.blob.lookupRetries=3`.
- Remove obsolete global/static properties `bootstrap.ledger.nodeAccounts.initialBalance=0` and `bootstrap.ledger.systemAccounts.initialBalance=0`.
- Remove obsolete `UsageEstimatorUtils` class.

**External impacts**:
Will cause a very small increase in fees charged for some `FileCreate` and `FileUpdate` operations.

### CODE REVIEW

File: hedera-node/src/test/java/com/hedera/services/legacy/unit/serialization/HFileMetaSerdeTest.java [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+	}
+
+	public static HFileMeta convert(FileInfo fi) throws DecoderException {
+		return new HFileMeta(
+				fi.getDeleted(),
+				JKey.mapKey(Key.newBuilder().setKeyList(fi.getKeys()).build()),
+				fi.getExpirationTime().getSeconds());
+	}
+
+	public static FileInfo toGrpc(HFileMeta info, FileID fid, long size) throws Exception {
```
Comment (anighanta) [inline hedera-node/src/test/java/com/hedera/services/legacy/unit/serialization/HFileMetaSerdeTest.java:214]: can we move this out of test code as we use this static method in `FileServiceHandler`

Comment (tinker-michaelj) [reply hedera-node/src/test/java/com/hedera/services/legacy/unit/serialization/HFileMetaSerdeTest.java:214]: `FileServiceHandler` is under _src/test/java_ also :)...it's just kept around to keep running a handful of legacy unit tests.

Once the properties refactor is complete, I believe we can delete a huge amount of these random legacy classes.

Comment (anighanta) [reply hedera-node/src/test/java/com/hedera/services/legacy/unit/serialization/HFileMetaSerdeTest.java:214]: Ahh.. missed that part. Makes sense

**Review by anighanta — APPROVED** [review]
LGTM

**Pull request merged**